executor/executor.go

```diff
@@ -427,10 +427,10 @@ type ShowDDLJobsExec struct {
 
 	jobNumber int
 	is        infoschema.InfoSchema
-	done      bool
 }
 
 // DDLJobRetriever retrieve the DDLJobs.
+// nolint:structcheck
 type DDLJobRetriever struct {
```

Comment:
Why this struct need "nolint..."?

Reply:
It has fields that it does not access by itself but used by other structs that embed it. Such as `ShowDDLJobsExec`. I think it is a false positive report.

Reply:
Got it.